addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/SiteIsolationScanRuleTest.java

```diff
@@ -36,6 +36,25 @@
 import org.zaproxy.addon.commonlib.CommonAlertTag;
 
 class SiteIsolationScanRuleTest extends PassiveScannerTest<SiteIsolationScanRule> {
+    @Test
+    void shouldRaiseCorpAlertGivenSiteIsNotIsolated() throws Exception {
+        // Given
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET / HTTP/1.1");
+        msg.setResponseHeader("HTTP/1.1 200 OK\r\n");
+        given(passiveScanData.isPage200(any())).willReturn(true);
```

Comment:
As you noticed elsewhere this is unnecessary because a simple status code check is done first.

Reply:
Removed this line

Reply:
Made some more changes, adding back a different mock. This conversation is irrelevant now.